pythonfmu/builder.py

```diff
@@ -36,6 +37,12 @@ def get_lib_extension() -> str:
     return platforms.get(platform.system(), "")
 
 
+def get_class_name(file_name: Path) -> str:
+    with open(str(file_name), 'r') as file:
+        data = file.read()
+        return re.search('class (.*)\(\s*Fmi2Slave\s*\)\s*:', data).group(1)
```

Comment:
```suggestion
        return re.search('^class (\w+)\(\s*Fmi2Slave\s*\)\s*:', data).group(1)
```

Reply:
`^` made it not match, but added your other suggestion.